CC/include/DistanceComputationTools.h

```diff
@@ -306,7 +307,11 @@ class CC_CORE_LIB_API DistanceComputationTools : public CCToolbox
 		\return negative error code or a positive value in case of success
 	**/
 	static int computeCloud2PlaneEquation(GenericIndexedCloudPersist* cloud, const PointCoordinateType* planeEquation, bool signedDistances = true, double * rms = nullptr);
+	
+	static int computeCloud2RectangleEquation(GenericIndexedCloudPersist *cloud, const PointCoordinateType widthX, const PointCoordinateType widthY, const SquareMatrix rotationTransform, const CCVector3 center, bool signedDist = true, double* rms = nullptr);
```

Comment:
`const PointCoordinateType widthX` -> no need for `const` on intrinsic type parameters (PointCoordinateType is either a float or double).

Is there a reason you aren't using pass-by-reference for the parameters I mentioned previously?